docs/environment/storage.md

```diff
@@ -76,3 +76,110 @@ Note that Redis and Memcache through ElastiCache run even when not used and thus
 pay-per-request mode where you don't pay when the store is not used. There is a package implementing the PSR cache interfaces using DynamoDB ([rikudou/psr6-dynamo-db](https://github.com/RikudouSage/DynamoDbCachePsr6)).
 
 This solution is ideal for cache data that can change during the life of the application (e.g. caching a website menu, an API response).
+
+#### Deploy DynamoDB table as part of serverless
+
+The example will be based on the minimal configuration which is as follows:
+
+```yaml
+service: app
+provider:
+    name: aws
+    runtime: provided.al2
+plugins:
+    - ./vendor/bref/bref
+functions:
+    app:
+        handler: index.php
+        layers:
+            - ${bref:layer.php-74-fpm}
+        events:
+            - httpApi: '*'
+```
+
+As any service, the DynamoDB table can be deployed using the `resources.Resources` subkey:
+
+```yaml
+service: app
+provider:
+    name: aws
+    runtime: provided.al2
+plugins:
+    - ./vendor/bref/bref
+functions:
+    app:
+        handler: index.php
+        layers:
+            - ${bref:layer.php-74-fpm}
+        events:
+            - httpApi: '*'
```

Comment:
Thanks! I think we can simplify the documentation by skipping the "standard" config (that will also be less confusing to integrate in Laravel or Symfony apps where serverless.yml may be a bit different):

```suggestion
service: app
...
```